slash/css/src/List/ContentItemMono/ContentItemMono.stories.ts

```diff
@@ -0,0 +1,195 @@
+import type { Args, Meta, StoryObj } from "@storybook/html";
+import "../../Card/Card.client.scss";
+import "../List.scss";
+import "./ContentItemMono.scss";
+
+const meta: Meta = {
+  title: "Client/Components/List/ContentItemMono",
+};
+
+export default meta;
+
+const template = ({ classModifier, ...args }: Args) => {
+  const container = document.createElement("div");
+
+  container.className = `af-content-item-mono${classModifier
+    .split(" ")
+    .filter(Boolean)
+    .map((modifier: string) => ` af-content-item-mono--${modifier}`)
+    .join("")}`;
+  container.className += ` af-content-item-mono--${args.size}`;
+
+  if (args.isDisabled) {
+    container.className += " af-content-item-mono--disabled";
+  }
```

Comment:
plus simple à lire et donc à suivre peut-être sous cette forme :
```typescript
const classes = container.classList;
classes.add("af-content-item-mono", `af-content-item-mono--${args.size}`);

classModifier
  .split(" ")
  .filter(Boolean)
  .forEach(classes.add);

if (args.isDisabled) {
  classes.add("af-content-item-mono--disabled");
}
...

Reply:
Je vais plutôt faire ça je pense : 
```
  const modifiers = [args.size, ...classModifier.split(" ")];

  if (args.isDisabled) {
    modifiers.push("disabled");
  }

  container.classList.add("af-content-item-mono");
  modifiers
    .filter(Boolean)
    .forEach((modifier: string) =>
      container.classList.add(`af-content-item-mono--${modifier}`),
    );```

Qu'en penses-tu ?

Reply:
ah oui, top. Peut-être encore mieux :
```typescript
const modifiers = [args.size, ...classModifier.split(" ")];

  if (args.isDisabled) {
    modifiers.push("disabled");
  }

  modifiers
    .filter(Boolean)
    .map((modifier) => `af-content-item-mono--${modifier}`);

  container.classList.add("af-content-item-mono", ...modifiers);
```

Reply:
Ca ne va pas fonctionner, map retourne le résultat, il ne remplacera pas la valeur de `modifiers`.
Comme ça je pense que ce sera mieux : 

```suggestion
  container.className = `af-content-item-mono${classModifier
  const modifiers = [args.size, ...classModifier.split(" ")];

  if (args.isDisabled) {
    modifiers.push("disabled");
  }

  container.classList.add(
    "af-content-item-mono",
    ...modifiers
      .filter(Boolean)
      .map((modifier) => `af-content-item-mono--${modifier}`),
  );
```

Reply:
je t'ai fait des suggestions sans compiler désolé, mais c'est exactement l'idée, bien vu ! 😆

Reply:
Resolved in https://github.com/AxaFrance/design-system/pull/436/commits/8dade79388edc689ea384444ffb60589b885b09c